fbfmaproom/fbflayout.py

```diff
@@ -403,32 +403,33 @@ def table_layout():
             dcc.Loading(
                 [
                     table.DataTable(
-                        id="summary",
+                        id="table",
                         page_action="none",
                         style_table={
-                            "height": "auto",
-                            "overflowY": "scroll",
+                            "height": "900px",
                         },
                         css=[
                             {
+                                # attempting to be more specific introduces a bizarre visual artifact
+                                # "selector": "div.dash-fixed-row tr:first-child",
```

Comment:
I tried uncommenting this line and commenting the one below, and I don't see any difference. Could you describe/screenshot what you're seeing?

It did put my browser tab into an infinite loop at one point, when I was resizing the window. I don't know if that's related to the problem you saw or not.

Reply:
OK, I tried it in firefox and the weird behavior is more pronounced.

The reason `div.dash-fixed-row` doesn't match is clear to me--no parent of the row in question has that class. The div that matches that pattern is the summary table (the fixed rows), whereas you're trying to hide the first non-fixed row.

Reply:
No that's incorrect. The first row of the fixed portion is the "built-in" header. DataTable prescribes that the first fixed row be the header which is formed from (in this case) the pandas column labels. There's no way to override this behavior, and we want the header to be the *last* row of the fixed portion. As such we hide the "true" header and insert our own. 

The problem is that if the first row of the fixed portion (that is to say the aforementioned "true" header) *and* the first row of the non-fixed/scrolling portion are not hidden, we get the weird visual artifacts I described.

Reply:
I see. I thought you were complaining that you tried a more specific selector and it didn't match for some reason that you couldn't figure out.

Now that I understand the situation, I think the comment could be clearer. e.g. "Hide the first fixed row and the first data row to work around glitchy browser behavior." And link to a github issue or forum discussion if you have one.

Reply:
When this line is uncommented and the one below it is commented. Only the "true" header is hidden. That would be desirable and would make duplicating the first row of the data portion unnecessary. Except it produces this result: 
![2022-03-25_09-26](https://user-images.githubusercontent.com/82181542/160129515-c3382447-bcd4-4ba3-a7a1-94016e25f07e.png)

The left border is missing and, while I can't show this in a static image, while scrolling the data portion, the header part will become unaligned by a single pixel.

Reply:
Ok, I'll update the comment. I couldn't find any forum or github discussion of this unfortunately.